SAMv1.tex

```diff
@@ -539,12 +540,66 @@ \subsection{The alignment section: mandatory fields}\label{sec:alnrecord}
   the next read. If {\sf PNEXT} is 0, no assumptions can be made on
   {\sf RNEXT} and bit 0x20.
 \item {\sf TLEN}: signed observed Template LENgth. If all segments are
-  mapped to the same reference, the unsigned observed template length
-  equals the number of bases from the leftmost mapped base to the
-  rightmost mapped base. The leftmost segment has a plus sign and the
-  rightmost has a minus sign. The sign of segments in the middle is
-  undefined. It is set as 0 for single-segment template or when the
-  information is unavailable.
+  mapped to the same reference sequence, the unsigned observed template length
+  equals the distance between the mapped end of the template and the
+  mapped start of the template, inclusively.\footnote{Thus a segment
```

Comment:
Should we get rid of a possible +1 ambiguity here by stating that a template with left-most base aligned at START and rightmost base aligned at END will have a TLEN of END - START + 1?

Reply:
Is that not what the maths already indicates?  Do you mean explicitly stating it again symbolically in addition to the numbers?  I'll give that a go.

Reply:
Note however your phrasing has locked it down to one of the two specifically implementations of TLEN in the wild. ;-)

Reply:
not really, since I was talking about templates, not reads....